packages/teleport/src/services/history.js

```diff
@@ -118,15 +118,7 @@ const match = url => route => {
 export default history;
 
 export function getUrlParameter(name, path) {
-  path = path || window.location.search;
-  const query = path.substring(1);
-  const vars = query.split('&');
-  for (var i = 0; i < vars.length; i++) {
-    var pair = vars[i].split('=');
-    if (decodeURIComponent(pair[0]) == name) {
-      return decodeURIComponent(pair[1]);
-    }
-  }
-
-  return '';
+  const params = new URLSearchParams(path || window.location.search);
```

Comment:
Lets remove `window.location.search` and good to go!

Reply:
revised